seriously-simple-podcasting.php

```diff
@@ -82,3 +86,10 @@
 	global $ssp_wp_rest_api;
 	$ssp_wp_rest_api = new Rest_Api_Controller( __FILE__, SSP_VERSION );
 }
+
```

Comment:
This can just be 
```
if ( ssp_is_elementor_ok()) {
	Main::instance();
}
```

Also, if we do the Main refactoring, we can just change it to

`$elementor_widgets = new Elementor_Widgets();`